masonry/src/contexts.rs

```diff
@@ -199,6 +199,26 @@ impl_context_method!(
     }
 );
 
+// Methods for all exclusive context types (i.e. those which have exclusive access to the global state).
+impl_context_method! {
+    AccessCtx<'_>,
+    ComposeCtx<'_>,
+    EventCtx<'_>,
+    LayoutCtx<'_>,
+    MutateCtx<'_>,
+    PaintCtx<'_>,
+    UpdateCtx<'_>,
+    {
+        /// Get the contexts needed to build and paint text sections.
+        ///
+        /// Note that in many cases, these contexts are.
```

Comment:
Text fragment

Reply:
I have no idea what I wanted to say here...